.hosted-stack-version

```diff
@@ -0,0 +1 @@
+9.1.0-48398db3
```

Comment:
I decided to use this file for now, we can talk about this implementation. For now, I'm just trying to test all the different scenarios